glue_job.tf

```diff
@@ -0,0 +1,233 @@
+resource "aws_glue_catalog_database" "catalog_db" {
+  name = "controlshift_${var.controlshift_environment}"
+}
+
+locals {
```

Comment:
What does locals do?

Reply:
defines local variables: https://www.terraform.io/docs/configuration/locals.html

Reply:
It's a local value (https://www.terraform.io/docs/configuration/locals.html). I think initially I was using this value in multiple places, but now it's only referenced once so could remove if it would be clearer. I personally find it clearer this way, where the variable name serves as documentation.